### ISSUE TITLE
ipyaladin 0.2.3 add_listener issue

### ISSUE THREAD

**Issue by RobinMeneust** [cds-astro/ipyaladin#67]
The click event doesn't seem to work in ver. 0.2.3, but it does in ver. 0.2.0.

In ver. 0.2.0, I get the following info box, but in ver. 0.2.3 the info box content remains empty
![image](https://github.com/cds-astro/ipyaladin/assets/85349121/c3bec629-67be-4746-a865-443ec0731143)

Notebook: https://github.com/cds-astro/ipyaladin/blob/master/examples/7_on-click-callback.ipynb
Environment: Python 3.10.6 and modules in [requirements.txt](https://github.com/cds-astro/ipyaladin/files/11815973/requirements.txt)

**Comment by ManonMarchand** [comment]
You're right. Thanks a lot for spotting this bug!

For reference during investivation: 

Console error when using the listenner : 

```
Uncaught TypeError: this is undefined
    model_events jupyter-aladin.js:263
    r aladin.js:2
    dispatch aladin.js:2
    handle aladin.js:2
```

**Comment by ManonMarchand** [comment]
It should be fixed with the version 0.2.4 that we just released. Don't hesitate to open an other issue if you find anything sketchy. 
Thanks again!
